src/components/about/ServiceTimes/ServiceTimeLocation.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import Image from 'next/image'
+import { Container } from '@/components/Container'
+import ClientImage from '@/components/ClientImage'
+
+interface ServiceTimeLocationProps {
+  time: string
+  location: string
+  imageSrc: string
+}
+const ServiceTimeLocation = ({ time, location,imageSrc }: ServiceTimeLocationProps) => {
+  
+  return (
+    <div className="relative bg-tertiary">
+      <Container className="relative">
+        <div className="flex py-20">
+          <div >
+            <div className="mb-4 text-2xl font-semibold uppercase">TIME</div>
```

Comment:
we already have `uppercase` so no need to capitalize.  Its either we do it in CSS or JS, but not in HTML